### DISCUSSION TITLE
Disable / Enable CapsLock Key

### DISCUSSION THREAD

**Discussion by JRKL75** [meetrevision/revision-tool#23] [Ideas]
Hi there. 

Any thoughts of implementing an option under "Usability" to disable the Capslock key?

Disable:
> Windows Registry Editor Version 5.00
> 
> [HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Keyboard Layout]
> "Scancode Map"=hex:00,00,00,00,00,00,00,00,02,00,00,00,00,00,3a,00,00,00,00,00

Enable:
delete the key: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Keyboard Layout\Scancode Map

Source: https://answers.microsoft.com/en-us/windows/forum/all/disable-caps-lock-key/8083d4ee-721d-463f-aeda-630cd7f047c3

I hope it's something that fits withing the vision of the tool.
Kind regards.

**Comment by melo936** [comment]
Thanks for the suggestion, it will be added.

Added in `2b68afce01d474abe919b1f298c11fe20992facf`
